GliaWidgets/Sources/Extensions/UIColor+Hex.swift

```diff
@@ -48,3 +48,9 @@ extension UIColor {
         }
     }
 }
+
+extension CGColor {
```

Comment:
I don't know if this is the right file to put this. No one would figure out that this extension is declared in an `UIColor` file.

Reply:
Fixed 28138aa40ffb224181edf1b751a612cf2c061f01